src/inc/crsttypes.h

```diff
@@ -236,6 +237,7 @@ int g_rgCrstLevelMap[] =
     3,			// CrstDynamicMT
     3,			// CrstDynLinkZapItems
     7,			// CrstEtwTypeLogHash
+    11,			// CrstEventPipe
```

Comment:
This is actually tool-generated code.  All of the lines in this array have tabs.  If we want to have spaces here, we probably need to do that as part of porting the tool to run in the coreclr repo.